torch/_functorch/aot_autograd.py

```diff
@@ -596,19 +597,29 @@ class ViewAndMutationMeta:
     # TODO: we should kill this
     # (need to default it to not break internal)
     is_train: bool = False
+    # Whether we are tracing through subclasses
+    # (ideally should remove this)
+    is_subclass: bool = False
 
     num_symints_saved_for_bw: Optional[int] = None
 
     def __post_init__(self):
         mutated_inp_indices = [
-            i for i, m in enumerate(self.input_info) if m.mutates_metadata or m.mutates_data
+            i for i, m in enumerate(self.input_info)
+            if m.mutates_data or m.mutates_metadata
```

Comment:
avoid needless reformatting like this, it makes review annoying. If you want to ufmt the file before working on it, do it in a separate PR